audb/core/load_to.py

```diff
@@ -322,15 +308,30 @@ def load_to(
     )
     if update:
         if only_metadata:
-            files = deps.attachment_files + deps.tables
+            files = deps.attachments + deps.tables
         else:
             files = deps.files
         for file in files:
             full_file = os.path.join(db_root, file)
             if os.path.exists(full_file):
-                checksum = audbackend.md5(full_file)
+                checksum = utils.md5(full_file)
                 if checksum != deps.checksum(file):
-                    os.remove(full_file)
+                    if os.path.isdir(full_file):
+                        for f in audeer.list_file_names(
+                            full_file,
+                            recursive=True,
+                            hidden=True,
+                        ):
+                            os.remove(f)
+                        for d in audeer.list_dir_names(
+                            full_file,
+                            recursive=True,
+                            hidden=True,
+                        ):
+                            os.rmdir(d)
+                        os.rmdir(full_file)
+                    else:
+                        os.remove(full_file)
```

Comment:
Maybe just use

```suggestion
                    audeer.rmdir(full_file)
```

Reply:
Nice, didn't know we have this function.